test/development/acceptance/hydration-error.test.ts

```diff
@@ -11,6 +11,11 @@ describe('Error overlay for hydration errors in Pages router', () => {
   const { next } = nextTestSetup({
     files: new FileRef(path.join(__dirname, 'fixtures', 'default-template')),
     skipStart: true,
+    // TODO: once Next.js minimal React version is 19.1, remove this override.
+    dependencies: {
+      react: isReact18 ? '^18.3.1' : '^19.1.0',
+      'react-dom': isReact18 ? '^18.3.1' : '^19.1.0',
```

Comment:
This shouldn't be needed. All tests should run with the latest React Canary installed.

Reply:
I noticed we're installing 19.0.0 when it's failing, not sure if it's caching or sth. We specified "^19.0.0" in the testing util, and it's not applying 19.1.0 when it was failing

Reply:
We prefer offline installs so that might be it. Either way, this should be hoisted into the test util since it is not compatible with the `NEXT_TEST_REACT_VERSION` environment variable.

Reply:
https://github.com/vercel/next.js/pull/77895